### PR TITLE
Add missing strings to Finnish translation

### PR DESCRIPTION
**Pull request by mattock**
Signed-off-by: Samuli Seppänen <samuli@openvpn.net>

### CODE REVIEW

File: res/openvpn-gui-res-fi.rc [hunk 1]
```diff
@@ -418,8 +440,18 @@ BEGIN
     IDS_ERR_WRITE_REGVALUE "Virhe kirjoitettaessa rekisterin arvoa HKEY_CURRENT_USER\\%s\\%s""."
 END
```
Comment (selvanair) [inline res/openvpn-gui-res-fi.rc]: delete this "END"

File: res/openvpn-gui-res-fi.rc [hunk 2]
```diff
@@ -418,8 +440,18 @@ BEGIN
...
 
+    /* importation */
+    IDS_ERR_IMPORT_EXISTS "Samanniminen (""%s"") asetustiedosto on jo olemassa."
+    IDS_ERR_IMPORT_FAILED "Asetustiedoston tuonti epäonnistui. Polkua ei voitu luoda:\n\n" \
+                          "%s\n\Varmista, että sinulla on tarvittavat tiedosto-oikeudet."
```
Comment (selvanair) [inline res/openvpn-gui-res-fi.rc]: \Varmista --> Varmista or \nVarmista

File: res/openvpn-gui-res-fi.rc [hunk 3]
```diff
@@ -418,8 +440,18 @@ BEGIN
     IDS_ERR_WRITE_REGVALUE "Virhe kirjoitettaessa rekisterin arvoa HKEY_CURRENT_USER\\%s\\%s""."
```
Comment (selvanair) [inline res/openvpn-gui-res-fi.rc]: Missing "" before HKEY_CURRENT_USER..

**Comment by mattock** [comment]
@selvanair all the issues have been fixed. I will test the translation myself on Windows 7 soon and try to get someone to do a language review.

**Pull request closed without merging**